tests/get-sample.test.js

```diff
@@ -0,0 +1,85 @@
+const test = require("ava");
+const getDB = require("../src/db");
+const app = require("../app");
+const http = require("http");
+const micro = require("micro");
+const listen = require("test-listen");
+const getJSON = require("bent")("json", "GET", 200, 400, 404);
+
+test("Get Sample", async (t) => {
+  const db = await getDB({ testDB: true });
+  const service = micro(app);
+  const url = await listen(service);
+
+  const objectToInsert = {
+    "summary_samples": {
+      "interface": {
+        "type": "image_classification",
+        "labels": [
+          {
+            "id": "valid",
+            "description": "valid"
+          },
+          {
+            "id": "invalid",
+            "description": "invalid"
+          }
+        ]
+      },
+      "summary": {
+        "samples": [
+          {
+            "hasAnnotation": false,
+            "version": 1
+          },
+          {
+            "hasAnnotation": false,
+            "version": 1
+          },
+          {
+            "hasAnnotation": false,
+            "version": 1
+          }
+        ]
+      }
+    },
+    short_id: "testSample",
+    samples: [
+      {
+        imageUrl:
+            "https://s3.amazonaws.com/asset.workaround.online/example-jobs/sticky-notes/image1.jpg",
+      },
+      {
+        imageUrl:
+            "https://s3.amazonaws.com/asset.workaround.online/example-jobs/sticky-notes/image2.jpg",
+      },
+    ]
+  };
+
+  db.prepare(
+      "INSERT INTO session_state (short_id, summary_samples) VALUES (?, ?)"
+  ).run(objectToInsert.short_id, JSON.stringify(objectToInsert.summary_samples));
+
+  const samplesQueries = []
+  objectToInsert.samples.forEach((sample, index) => {
+    samplesQueries.push(
+        db.prepare("INSERT INTO sample_state (session_short_id, session_sample_index, content) VALUES (?, ?, ?)")
+            .run(objectToInsert.short_id, index, JSON.stringify(sample))
+    )
+  })
+  await Promise.all(samplesQueries)
+
+  const response = await getJSON(`${url}/api/session/${objectToInsert.short_id}/sample/0`);
+  t.assert(response);
+  console.log(response)
+
+  t.deepEqual(response, {
+    imageUrl:
+        "https://s3.amazonaws.com/asset.workaround.online/example-jobs/sticky-notes/image1.jpg",
+    annotation: null
+  })
+
+  db.prepare(`DELETE FROM session_state WHERE short_id = ?`).run(
+      objectToInsert.short_id
```

Comment:
i don't think we need to worry about cleanup because of the parameter passed to `getDB({ testDB: true })`. When `testDB === true` it should automatically clean up.